### ISSUE TITLE
🐛 [BUG] - Bliss OS 16 v16.9.7-gapps-20240911 Kernel SU Superuser Root Access is Broken

### ISSUE THREAD

**Issue by TheFlash12314** [BlissRoms-x86/support#99]
### Description

Kernel SU root access is broken on latest build Bliss-v16.9.7-x86_64-OFFICIAL-gapps-20240911.
Kernel SU has given superuser access but when enter the app do not have root access.

 Tried testing using build Bliss-v16.9.6-x86_64-OFFICIAL-gapps-20240602 was working perfectly fine.

![Screenshot_20241007-165534_KernelSU](https://github.com/user-attachments/assets/b94f3b24-debf-4b04-96d9-66f4f5450c17)
![Screenshot_20241007-165546_KernelSU](https://github.com/user-attachments/assets/1ec9b38f-a767-4880-9196-e979fe28bb6e)
![Screenshot_20241007-165919_Termux](https://github.com/user-attachments/assets/4c861954-28ba-41a8-96cb-26763ded20a9)

### Reproduction steps

```bash
1. Install latest bliss OS v16.9.7-gapps-20240911.
2. Give Termux superuser root access.
3. Launch Termux and enter code "sudo su"/"tsu".
```

### Screenshots

```bash
![Screenshot_20241007-165534_KernelSU](https://github.com/user-attachments/assets/b94f3b24-debf-4b04-96d9-66f4f5450c17)
![Screenshot_20241007-165546_KernelSU](https://github.com/user-attachments/assets/1ec9b38f-a767-4880-9196-e979fe28bb6e)
![Screenshot_20241007-165919_Termux](https://github.com/user-attachments/assets/4c861954-28ba-41a8-96cb-26763ded20a9)
```

### Category

Other

### OS Version

16.x

**Comment by hmtheboy154** [comment]
Bugs has been found and a new build with a temp fix will be released soon
https://github.com/tiann/KernelSU/issues/2113

**Comment by Sheepmister** [comment]
> Bugs has been found and a new build with a temp fix will be released soon [tiann/KernelSU#2113](https://github.com/tiann/KernelSU/issues/2113)

hi sir i know this is old but can i ask what is the stable bliss os version i dont care about old kernel as long as it works with root so i can install gearlock android 11/12 version(not beta)

**Comment by buihaiminh1301-afk** [comment]
try /system/bin/su
